public/controllers/agent/agents.js

```diff
@@ -309,6 +311,19 @@ export class AgentsController {
       );
     };
 
+    /**
+     * This adds timezone offset to a given date
+     * @param {String} binding_text
+     * @param {String} date
+     */
+    this.$scope.offsetTimestamp = (text, time) => {
```

Comment:
Very similar to the last comment, now, you still need to wrap to the scope, it's okay but this time move the logic to an instance method, then just wrap it using `this.$scope.offsetTimestamp = (text, time) => this.offsetTimestamp(text, time);`